package.json

```diff
@@ -102,16 +102,29 @@
     "extends": [
       "config:base"
     ],
-    "ignoreDeps": [
-      "prettier"
-    ],
     "lockFileMaintenance": {
       "enabled": true
     },
-    "pinVersions": false,
-    "prCreation": "not-pending",
+    "ignoreDeps": [
+      "prettier"
+    ],
     "semanticCommits": true,
-    "semanticPrefix": "chore:"
+    "semanticCommitScope": "{{parentDir}}",
```

Comment:
I suppose this will not work, as dependencies can be updated in multiple packages and you don't want to have separate commits / PRs, because then your dependencies will run out of sync, which might cause trouble for some dependencies.

Reply:
As this appears to be a rather straightforward/standard use case, I would expect renovate to be able to deal with this. If it turns out to be incapable of this, I would not mind getting rid of this setting again.